test/rpc/test_data/get_events.py

```diff
@@ -91,27 +93,6 @@
     "chunk_size": 10,
 }
 
-BLOCK_FROM_0_TO_LATEST_CHUNK_SIZE_1 = {
```

Comment:
Thank you for removing at least a part of these inconvenient constants xD

Reply:
I think I can change the method creating the `filter` parameter to have all the parameters needed to create those constants, what do you think?